### ISSUE TITLE
How to install Numverify API KEY

### ISSUE THREAD

**Issue by vicmac-github** [sundowndev/phoneinfoga#994]
![Phoneinfoga](https://user-images.githubusercontent.com/69369181/151239393-4b589b19-c0a9-4357-8815-23b2e51b3291.jpg)

I already have an API KEY of numverify, but i dont know where to put the code.

**Comment by sundowndev** [comment]
Hi, you have to specify the API key as a env var when launching the scan or the HTTP server.

Example : 

```
$ NUMVERIFY_API_KEY=secret phoneinfoga serve
```

**Comment by brickjawn** [comment]
awesome! thanks

**Comment by dwbravo** [comment]
Hi, I am having the same issue after reading this it helped out, but it still did not work i think missing something could you dumb it down just a bit.

**Comment by alan6t9** [comment]
same issue

**Comment by AzamServer** [comment]
How would you do this in the docker compose file?

**Comment by sundowndev** [comment]
> How would you do this in the docker compose file?

https://docs.docker.com/compose/environment-variables/

**Comment by AzamServer** [comment]
> > How would you do this in the docker compose file?
> 
> https://docs.docker.com/compose/environment-variables/

Getting this error:
`ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.environment: 'NUMVERIFY_API_KEY'`

**Comment by Imesh2042** [comment]
can you please explain more, i dont even know what is environmental variable

**Comment by Darkcast** [comment]
why not just have a simple config file ?

**Comment by Recent-Salt** [comment]
@Darkcast Where do I create the config file to include the API key, so phoneinfoga uses it automatically? 
I know, silly question, I am beginner. Not using docker, but WSL2 and Kali Linux.

**Comment by sundowndev** [comment]
Phoneinfoga doesn't support config files yet. We could easily add this by supporting .env files. Please open another issue for this.

**Comment by SecuritySphinx** [comment]
when I try to run Numverify it gives me You cannot consume this service
